stdlib/ref.ml

```diff
@@ -0,0 +1,35 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                        Nicolas Ojeda Bar, LexiFi                       *)
+(*                                                                        *)
+(*   Copyright 2020 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+type 'a t = 'a ref = {mutable contents : 'a}
```

Comment:
Would `[@@unboxed]` be useful here?

CC @c-cube

Reply:
"unboxable record types are the immutable record types that have only one field" ([ref](https://caml.inria.fr/pub/docs/manual-ocaml/intfc.html#ss:c-tuples-and-records)) – I guess not.